spring-cloud-stream-binder-rabbit/src/main/java/org/springframework/cloud/stream/binder/rabbit/RabbitMessageChannelBinder.java

```diff
@@ -62,9 +65,6 @@
 import org.springframework.util.Assert;
 import org.springframework.util.StringUtils;
 
-import com.rabbitmq.client.AMQP;
-import com.rabbitmq.client.Envelope;
```

Comment:
We probably have to address this as soon as possible, because there is going to be a mess all the time when we with Gary interfere to the code.

The SF imports order: https://github.com/spring-projects/spring-framework/wiki/Spring-Framework-Code-Style#import-statements